frontend/src/common/LoginController.ts

```diff
@@ -39,7 +39,7 @@ export default class LoginController {
     }
 
     checkSavedUser() {
-        const savedUser = localStorage.getItem(this.savedUserKey)
+        const savedUser = localStorage.getItem(this.dbKey)
         if (!savedUser) {

```

Comment:
Adding transactions saves the to the db not to the user, so retrieving the user works only if i get it from the db